packages/contract-processors/src/moonbeam.ts

```diff
@@ -317,7 +340,10 @@ const CallProcessor: SecondLayerHandlerProcessor<SubqlHandlerKind.Call, Moonbeam
       const [tx] = input.extrinsic.method.args as [EthTransaction];
 
       // if `to` is null then we handle contract creation
-      if ((ds.address && !stringNormalizedEq(ds.address, to)) || (ds.address === null && !tx.action.isCreate)) {
+      if (
+        (ds.processorOptions?.address && !stringNormalizedEq(ds.processorOptions.address, to)) ||
+        (ds.processorOptions?.address === null && !tx.action.isCreate)
```

Comment:
Off the topic,but I'm thinking we can skip contract creation for now. (filter them out)
Contract creation can be left to the dynamic ds task.